### PR TITLE
PMM-2888 (subtask PMM-3131) documenting text file collector usage

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-238/

File: doc/source/pmm-admin.rst [hunk 1]
```diff
@@ -340,6 +340,38 @@ For more information, run
...
+``node_exporter`` metrics out of the box. There is a simple method to extend the
+list of available metrics without modifying the ``node_exporter`` code. It is
+based on the textfile collector.
+
+Starting from version 1.16.0, this collector is enabled for the
+``linux:metrics`` in |pmm-client| by default.
+
+The default directory for reading text files with the metrics is
+``/usr/local/percona/pmm-client/textfile-collector``, and the exporter reads files
+from it with the `.prom` extension.
```
Comment (michaelcoburn) [inline doc/source/pmm-admin.rst]:
```suggestion
from it with the `.prom` extension.  You are responsible for running a cronjob or other regular process to generate the metric series data and write it to this directory.
```

File: doc/source/pmm-admin.rst [hunk 2]
```diff
@@ -340,6 +340,38 @@ For more information, run
...
+
+The most obvious way to fill this text file with the needed data is to add a
+``crontab`` task, which would collect metrics and place them into a file.
+
```
Comment (michaelcoburn) [inline doc/source/pmm-admin.rst]: Example - Collecting docker container information
This example will show you how to collect the number of running and stopped docker  containers on a host::

File: doc/source/pmm-admin.rst [hunk 3]
```diff
@@ -340,6 +340,38 @@ For more information, run
...
+Here are the ``cron`` commands example for collecting the number of running and
```
Comment (michaelcoburn) [inline doc/source/pmm-admin.rst]:
```suggestion
```

File: doc/source/pmm-admin.rst [hunk 4]
```diff
@@ -340,6 +340,38 @@ For more information, run
...
+stopping docker containers::
```
Comment (michaelcoburn) [inline doc/source/pmm-admin.rst]:
```suggestion
```

File: doc/source/pmm-admin.rst [hunk 5]
```diff
@@ -340,6 +340,38 @@ For more information, run
...
+
+  */1 * * * *     root   echo -n "" > /tmp/docker_all.prom; /usr/bin/docker ps -a | sed -n '1!p'| /usr/bin/wc -l | sed -ne 's/^/node_docker_containers_total /p' >> /usr/local/percona/pmm-client/docker_all.prom;
+  */1 * * * *     root   echo -n "" > /tmp/docker_running.prom; /usr/bin/docker ps | sed -n '1!p'| /usr/bin/wc -l | sed -ne 's/^/node_docker_containers_running_total /p' >>/usr/local/percona/pmm-client/docker_running.prom;
+
```
Comment (michaelcoburn) [inline doc/source/pmm-admin.rst]: Add an example of the output, should look like this:

node_docker_containers_total 2
node_docker_containers_running_total 1

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-238/

**Pull request merged**